stripe/src/main/java/com/stripe/android/paymentsheet/model/ClientSecret.kt

```diff
@@ -0,0 +1,23 @@
+package com.stripe.android.paymentsheet.model
+
+import android.os.Parcelable
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.SetupIntent
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Represents the client secret for a [SetupIntent] or [PaymentIntent]
+ */
+internal interface ClientSecret : Parcelable {
```

Comment:
FYI there's also `PaymentIntent.ClientSecret`.

Reply:
Yes, that's just to validate and extract the id from the client secret, which we'll eventually remove. I kept it as is because it's an internal class and its use is not too widespread. I could also turn that into just a method.